src/main/java/roomescape/exeption/GlobalExeptionHandler.java

```diff
@@ -0,0 +1,14 @@
+package roomescape.exeption;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
```

Comment:
`@ControllerAdvice`에 대해 알아보면 좋을거 같아요
```suggestion
import org.springframework.web.bind.annotation.ControllerAdvice;

@ControllerAdvice
```